### DISCUSSION TITLE
Make Fancy Skulls setting apply to Enderman Head from Hardcore Ender Expansion?

### DISCUSSION THREAD

**Discussion by Rozmir-Rohi** [Roadhog360/Et-Futurum-Requiem#374] [Improvements]
Currently the Enderman head is 2D when the fancy skulls setting is enabled.

**Comment by xronin01** [comment]
Fancy skulls also don't work with the skulls from IguanaTweaksTConstruct and Witchery

**Comment by Roadhog360** [comment]
Because each one would require individual support and a custom model code. This isn't planned, I'd rather pick up Headcrumbs honestly
